checkov/terraform/checks/graph_checks/aws/KmsKeyWildcardPrincipal.yaml

```
@@ -0,0 +1,28 @@
+---
+metadata:
+  id: "CKV2_AWS_67"
```

```suggestion
  id: "CKV_AWS_33"
```
no need to create a new ID

---

Id change made